src/components/TopBar/components/Menu/Menu.scss

```diff
@@ -2,7 +2,10 @@
 
 $activator-variables: (
   min-width: rem(36px),
-  transition: background-color 0.1s,
+  transition: (
+    duration(fast) background-color,
```

Comment:
Should background color have easing as well?

Reply:
Actually, let me remove the changes I made to this map. The color transition is applied to children, not on the top level activator element.